### PR TITLE
Try to kill notifications for dead threads

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@andrewconner maybe this will do what we want?

### CODE REVIEW

**Comment by andrewconner** [comment]
Sweet, will review.

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala [hunk 1]
```diff
@@ -245,8 +246,8 @@ class NotificationDeliveryCommander @Inject() (
     sendPushNotification(userId, MessageCountPushNotification(totalUnreadCount))
   }
 
-  def notifyRemoveThread(userId: Id[User], keepId: PublicId[Keep]): Unit =
-    sendToUser(userId, Json.arr("remove_thread", keepId))
+  def notifyRemoveThread(userId: Id[User], keepId: Id[Keep]): Unit =
+    sendToUser(userId, Json.arr("remove_thread", Keep.publicId(keepId)))
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:250]: Would it make sense to open this to any notification? Send the thread id (in this case, a keep id, but could be a regular notification id as well)

**Comment by andrewconner** [comment]
Thanks, looks good. Minor comment above, otherwise merge when you're happy. I'm doing the extension side of this now.

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:250]: I know it's a different model on the backend, but when a keep is deleted, is the "new keep" notification removed already? Not sure I checked that one. It's zero extra work from this to get the ext to support deleting that guy as well.

Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:250]: Sure, I can add notification deletion too.

Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:250]: Could it be the same message to clients?

Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:250]: `["remove_notification", id]`?

Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:250]: Then it'll be symmetric with how you send notifications

Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:250]: Yep, sounds good. I'll tag you in it when I'm ready.

**Pull request merged**